### PR TITLE
added about page

### PR DESCRIPTION
**Pull request by jacobj10**
this is a temp version of the about page, we need to come up with a standardized way of differentiating between team, discussed w @ehsanmasdar and @mjkaufer. This should be good for now

### CODE REVIEW

**Review by kexin-zhang — APPROVED** [review]
Looks good to me. We can make improvements to the page after we get an initial version of the site up.

**Review by ehsanmasdar — CHANGES_REQUESTED** [review]
The hexagons on the about page are warping in weird ways

**Comment by jacobj10** [comment]
@kexin-zhang @mjkaufer @ehsanmasdar this should be the updated version of the about page that we want

**Comment by mjkaufer** [comment]
There was a misalignment in the about page (due to bad css on my part) that I just fixed, I agree with the css hexagon thing (also my fault) but I can adjust the hexagon css accordingly once we get real team photos.

**Review by mjkaufer — APPROVED** [review]
beautiful, jekyll himself would be proud

File: _data/members.csv [hunk 1]
```diff
@@ -1,16 +1,11 @@
...
-4/10/2017 20:08:09,Rohun Atluri,,Ops,http://rohunatluri.com/images/profile_picture.jpg,No,"I'm a second year CS undergrad. I love spending time outside - playing soccer, hiking, backpacking, snowboarding.",facebook,github,linkedin,link,https://www.facebook.com/rohun.atluri,https://github.com/rohunatluri,https://www.linkedin.com/in/rohun-atluri-581787a5/,http://rohunatluri.com
-2/13/2017 1:54:33,Ryan Petschek,,Tech,https://avatars1.githubusercontent.com/u/1111373?v=3&s=460,No,"Originally from Connecticut, I'm a first-year computer science student at Georgia Tech.",github,globe,envelope,circle,https://github.com/petschekr,https://ryanpetschek.me/,mailto:petschekr@gmail.com,https://keybase.io/petschekr
-2/13/2017 0:44:09,Siwei Li,Robert,Tech,http://blog.wayriad.org/img/person/myphoto.jpg,No,"Learn, make, and serve, humbled yet motivated - a Georgia Tech CS Undergrad who knows how to make things :)",github,linkedin,envelope,link,https://github.com/RSLi,https://www.linkedin.com/in/robertsiweili,mailto:robertsli@163.com,http://blog.wayriad.org/
\ No newline at end of file
+name
```
Comment (kexin-zhang) [inline _data/members.csv:1]: do we still need this file if we have a csv file for each team?

Comment (jacobj10) [reply _data/members.csv:1]: kept for posterity, will remove

**Comment by ehsanmasdar** [comment]
We might want to hide this page until we get real pictures. Maybe replace it with a blurb about the overall org? "HackGT is an org that does xyz"

**Comment by kexin-zhang** [comment]
I found a couple of small-ish issues while playing around with adding real content.
1. ~~For team descriptions, if the description has a comma in it, it gets truncated where the comma is. (see screenshot below, there's supposed to a comma and more words after 'marketing').
To replicate, try setting this as the description for comms: 
`The communications team serves as the voice for HackGT. Through marketing, brand management, social media engagement, press releases, etc., this team handles all forms of communication between the world and the organization.`~~
**EDIT**:  found why this was happening and pushed a quick solution.
2. When the descriptions are longer, the gradient for the team title ('communicators' in this case) is off - see screenshot below.
3. We should adjust hexagons for 1:1 dimension images 

image

**Review by kexin-zhang — CHANGES_REQUESTED** [review]
requesting changes b/c of my previous comment

**Review by mjkaufer — CHANGES_REQUESTED** [review]
jk the hexagons scale like trash when there are lots of members now for some reason, I will tryna fix this

**Comment by mjkaufer** [comment]
Also for the hexagons that display in the about page, they have to get very small in order to fit everyone. There are a couple of different ways we could solve this

1. @ehsanmasdar suggests tiling them in a non-random manner (connect edges of hexagons), and don't worry if the tiling goes off of the edge. To avoid some team members never getting face time, sort the order before tiling
2. We could also just only display 5 random members at a time, if we really want the random positioning to still be a factor.

**Comment by mjkaufer** [comment]
I removed the hexagons from about page, eventually they will come back [I promise](https://github.com/HackGT/hackgt-org-website/issues/44)

@ehsanmasdar @kexin-zhang

**Review by mjkaufer — APPROVED** [review]

**Review by kexin-zhang — APPROVED** [review]
Bump, we're hiding the picture hexagons for now so that we can push something to the about page soon.

**Review by ehsanmasdar — APPROVED** [review]

**Pull request merged**